src/functions/depsUtils.ts

```diff
@@ -6,7 +6,16 @@ export const getLicense = (dep: Dependency, overrideLicense?: Config["overrideLi
     if (result) return result;
   }
   if (typeof dep.license === "string") return dep.license;
+  // Some old packages used license objects or licenses property containing an array.
+  // ref. https://docs.npmjs.com/cli/v10/configuring-npm/package-json#license
   if (typeof dep.license === "object") return dep.license.type;
+  if (Array.isArray(dep.licenses)) {
+    if (dep.licenses.length > 1) {
+      const joined = dep.licenses.map((l) => l.type).join(" OR ");
+      return `(${joined})`;
+    }
+    return dep.licenses.at(0)?.type ?? "";
+  }
```

Comment:
If there are elements without a type, could you treat the entire retrieval as a failure and return `""`?

The licenses specification is vague, and there's no guarantee that `type` will always be present.
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#license

If none of the elements have a `type`, a string like " OR OR OR " might be generated. However, if elements without `type` are excluded, there's a risk that a string like "MIR OR ISC" could be generated when "MIT OR ISC OR Apache" is expected.

It might be safer to make the function work only when all elements in the array have a `type`.